packages/theme-provider/test/index.tsx

```diff
@@ -15,7 +14,7 @@ const renderJSON = el => renderer.create(el).toJSON()
 
 test('renders', () => {
   const json = renderJSON(
-    <ThemeProvider>
+    <ThemeProvider theme={{}}>
```

Comment:
Okay, the tests in other packages would also need this fix.
There are currently 7 failing on my machine before rebasing this branch on top of fresh master and 4 after rebase.

@jxnblk This is public API change. Does it make sense to use ThemeProvider without passing theme?

Reply:
That's a good question. There *is* the notion of a default theme, which is currently hard-coded (in various places knowing me) that includes `theme.breakpoints` and `theme.fontSizes`, but that currently doesn't require the `ThemeProvider` the way it's implemented. I think it's worth considering changing how that works – in #832 I suggested removing that altogether, in which case making the `theme` prop required, probably makes sense here

Reply:
I'll fix that unit test then and we'll make it required.